collective/easyform/profiles/default/actions.xml

```diff
@@ -42,10 +42,13 @@
    <property name="visible">True</property>
   </object>
   <object name="Fields" meta_type="CMF Action" i18n:domain="collective.easyform">
-    <property name="title" i18n:translate="">Fields</property>
+    <property name="title" i18n:translate="">Define form fields</property>
     <property name="description" i18n:translate=""></property>
+    <!-- For some reason Zope is considering Fields & Actions views as canonical objects -->
+    <!-- The condition below works around this by building 1) an absolute url for the form -->
+    <!-- Or 2) a relative one when user is not at canonical view -->
     <property
-      name="url_expr">string:${object_url}/fields</property>
+      name="url_expr">python:object_url + '/fields' if context.restrictedTraverse('@@plone_interface_info').provides('collective.easyform.interfaces.IEasyForm') else './fields'</property>
     <property name="icon_expr"></property>
     <property
       name="available_expr">python:object.portal_type == 'EasyForm'</property>
```

Comment:
This should be enough, however when inside `/form/fields` or `/form/actions`, Acquisition is inheriting portal_type from the `form` parent, thus making the condition falsely evaluate to True. I think `python:hasattr(aq_base(object), 'portal_type')` would fix it but `aq_base` isn't available in expressions. Is there a better fix? /cc @smcmahon